lib/hnscan.js

```diff
@@ -423,8 +423,9 @@ class Hnscan extends EventEmitter {
 
       peers.push({
         id: peer.id,
-        addr: peer.fullname(),
-        addrlocal: !peer.local.isNull() ? peer.local.fullname() : undefined,
+        addr: peer.address.hostname,
```

Comment:
Yo @kilpatty , is this right? I dug into what was actually being returned by peers, and .address seemed like the only ip-like structure and was under this class:

```

<NetAddress: hostname=74.207.247.120:12038 services=11 date=2020-03-06T17:29:29Z>
```

Reply:
LGTM -> Can also use peer.hostname() as a short cut to peer.address.hostname, but this works as well.